src/SimpleDetailsList.tsx

```diff
@@ -1,21 +1,23 @@
 import * as React from 'react';
+import { CommandBar } from 'office-ui-fabric-react/lib/CommandBar';
+import {
+  ContextualMenu,
+  DirectionalHint,
+  IContextualMenuItem,
+  IContextualMenuProps
+} from 'office-ui-fabric-react/lib/ContextualMenu';
+import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';

```

Comment:
This sample uses webpack for simplicity, but our app has its own bundling, so we are planning to deliver the office-ui-fabric-react.min.js by now. I already bundled a .d.ts file like the ones in Typings for your library so we can use TypeScript with that catch-all JS.